ElectronicObserverTypes/Extensions/ShipDataExtensions.cs

```diff
@@ -278,19 +278,22 @@ EquipmentId.RadarLarge_15mDuplexRangefinder_Type21AirRADARKai2 or
 		.Count(e => e?.MasterEquipment.IsSonar() is true) >= count;
 
 	public static bool HasAntiSubmarineAircraft(this IShipData ship, int count = 1) => ship.AllSlotInstance
-		.Count(e => e?.MasterEquipment.IsAntiSubmarineAircraft is true) >= count;
+		.Zip(ship.Aircraft, (e, size) => (e, size))
```

Comment:
Tuple element names should be pascal case. Tuples are essentially just anonymous structs.